### PR TITLE
Adds the ability to create a cycle of values in the test plan.

### PR DESCRIPTION
**Pull request by huitseeker**

### CODE REVIEW

**Comment by huitseeker** [comment]
screen shot 2015-06-18 at 20 02 02
Produces a nice 'bump'

**Comment by dragos** [comment]
Nice graphs!

**Comment by skyluc** [comment]
I think it needs a bit more doc, to explain what is the expected result.
I had trouble to get it to push anything else than the first value while trying it at first. It has to have a rate of more than 100 value/s to start to the see the 2nd value, more than 200 values/s to start to see the 3rd one, etc.

**Comment by huitseeker** [comment]
@skyluc Thanks ! This is in fact a bug and I believe I just fixed it :)

File: README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ sequence = [
     rate = 4
     duration = 1
   }
+  { type = cycle        # cycle through 'values' periodically
```
Comment (skyluc) [inline README.md]: I don't think `periodically` makes sense here. It means 'at regular interval'. 

`cycle though the 'value'` should be good enough.

**Comment by skyluc** [comment]
Nitpicking, otherwise LGTM.

**Comment by huitseeker** [comment]
Fixed!

**Pull request merged**